Mixpanel/Mixpanel.m

```diff
@@ -838,26 +849,33 @@ - (NSDictionary *)collectAutomaticProperties
                                   @"mp_lib": @"iphone",
                                   @"$lib_version": [self libVersion],
                                   @"$manufacturer": @"Apple",
+#if !defined(MIXPANEL_WATCH_EXTENSION)
                                   @"$os": [device systemName],
                                   @"$os_version": [device systemVersion],
+                                  @"$screen_height": @((NSInteger)size.height),
+                                  @"$screen_width": @((NSInteger)size.width),
+#endif
                                   @"$model": deviceModel,
                                   @"mp_device_model": deviceModel, //legacy
-                                  @"$screen_height": @((NSInteger)size.height),
-                                  @"$screen_width": @((NSInteger)size.width)
                                   }];
+#if defined(MIXPANEL_WATCH_EXTENSION)
+    [p addEntriesFromDictionary:[MixpanelWatchProperties collectAutomaticProperties]];
+#endif
     return [p copy];
 }
 
 + (BOOL)inBackground
 {
-#if !defined(MIXPANEL_APP_EXTENSION)
+#if defined(MIXPANEL_WATCH_EXTENSION)
+    return [WKExtension sharedExtension].applicationState == WKApplicationStateBackground;
```

Comment:
oh i didn't realize that was 3.0+. looks like it's only used in A/B testing so i'll remove this